schemas/com.roku/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a Roku video player event",
+  "self": {
+    "format": "jsonschema",
+    "name": "video",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "string",
+      "description": "ID generated when video tracking of the video node was initialized.",
+      "maxLength": 255
+    },
+    "contentId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Played video content ID.",
+      "maxLength": 255
+    },
+    "contentTitle": {
+      "description": "Title of the played content.",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 65535
+    },
+    "contentType": {
+      "description": "Type of the played content.",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 255
+    },
+    "streamFormat": {
+      "description": "Type of content (e.g., mp4, wma, mkv).",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 255
+    },
+    "streamUrl": {
+      "type": "string",
+      "description": "URL of the current stream.",
+      "maxLength": 65535,
+      "format": "uri"
+    },
+    "measuredBitrate": {
+      "type": "integer",
+      "description": "Measured bitrate (bps) of the network when the stream was selected.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "streamBitrate": {
+      "type": "integer",
+      "description": "Current bitrate of the stream.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "isUnderrun": {
+      "type": "boolean",
+      "description": "Indicates whether the stream was downloaded due to an underrun."
+    },
+    "isResumed": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "Indicates whether the playback was resumed after trickplay."
+    },
+    "videoFormat": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Format of the currently playing video stream (e.g., hevc, mpeg2).",
+      "maxLength": 65535
+    },
```

Comment:
So streamFormat is the container and videoFormat is the codec?

Reply:
Yes, this is true. There is also a difference in that videoFormat is automatically inferred while streamFormat is passed in content metadata to the video player node.